tools/tcpconnect.py

```diff
@@ -324,15 +492,21 @@ def depict_cnt(counts_tab, l3prot='ipv4'):
         print("%-9s" % ("TIME(s)"), end="")
     if args.print_uid:
         print("%-6s" % ("UID"), end="")
-    print("%-6s %-12s %-2s %-16s %-16s %-4s" % ("PID", "COMM", "IP", "SADDR",
-        "DADDR", "DPORT"))
+    print("%-6s %-12s %-2s %-16s %-16s %-6s" % ("PID", "COMM", "IP", "SADDR",
+        "DADDR", "DPORT"), end="")
+    if args.dns:
+        print("%-10s %-20s" % ("DNS_DELTA", "DNS_QUERY"))
```

Comment:
Other columns don't include "_". I'd rename these to "DNSus" (and don't include the "us" in the column rows) and "QUERY".

QUERY should also be "%s": it doesn't need the 20-char whitespace padding, it can grow/shrink as needed (indeed, I put such "%s"'s last so they don't unalign other columns).

Reply:
i updated it to DNS(ms) and explained it.  Fixed the query printing.